CHANGES.txt

```diff
@@ -27,6 +27,11 @@ Breaking Changes
 Changes
 =======
 
+ - Added Privileges plugin for the Enterprise Edition.
```

Comment:
This is about adding a privileges tab to the admin ui, right? We could mention that in the change entry.